### DISCUSSION TITLE
Default Output Format

### DISCUSSION THREAD

**Discussion by kmgill** [kmgill/mars-raw-utils#48] [General]
I am considering making the default output format tifs rather than pngs. Any opposition? This can be done in the `sciimg` codebase, though part of this would be to allow projects to change it during build time. 

As of right now, the output format can be set using an environment variable like one of the following:
```
export MARS_OUTPUT_FORMAT=tif
export MARS_OUTPUT_FORMAT=jpg
export MARS_OUTPUT_FORMAT=png
export MARS_OUTPUT_FORMAT=dng
```

**Comment by sschmaus** [comment]
Sounds good to me because I had problems with the 16bit pngs being unsupported in other programs in the past. 
Ideally the tifs would also default to some lossless compression so filesizes don't get unnecessarily huge.
